src/camvidlog2/yoloe/plot.py

```
@@ -0,0 +1,48 @@
+from collections.abc import Generator, Iterator
+
+import numpy as np
+import pandas as pd
+import supervision as sv
+
+
+def overlay_detections(
+    frames: Iterator[np.ndarray], detections: pd.DataFrame
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)